scripts/firedrake-install

```diff
@@ -1266,18 +1266,17 @@ def build_and_install_libsupermesh():
             check_call(["git", "clean", "-f", "-x", "-d"])
             check_call(["mkdir", "-p", "build"])
             with directory("build"):
-                cmd = ["cmake", "..", "-DBUILD_SHARED_LIBS=ON",
-                       "-DCMAKE_INSTALL_PREFIX=" + firedrake_env,
-                       "-DMPI_C_COMPILER=" + cc,
-                       "-DMPI_CXX_COMPILER=" + cxx,
-                       "-DMPI_Fortran_COMPILER=" + f90,
-                       "-DCMAKE_Fortran_COMPILER=" + f90,
-                       "-DLIBSUPERMESH_AUTO_COMPILER_FLAGS=OFF"]
-                fortran, version = sniff_compiler(f90)
-                if fortran == "GNU" and version >= Version("10"):
```

Comment:
I assume this is safe to remove?

Reply:
This is already in the libsupermesh `CMakeLists.txt`